MCprep_addon/conf.py

```diff
@@ -70,6 +70,12 @@ def init():
 		"MCprep_resources",
 		"mcprep_data_update.json")
 
+	global optimizer_json_path 
```

Comment:
Where does this get used? I recall our discussion of this being used as an override, but I'm not seeing where this gets picked up in the code.

Reply:
Whoops, that was back when I was thinking of adding JSON configs to the optimizer (which now uses the new advanced panel). Thanks for catching that

Reply:
Sure thing! Reminder to delete when you get to updating this next :)